### PR TITLE
Add MEDSEA_ANALYSIS_FORECAST_PHY_006_013 in the URL normalizer

### PR DESCRIPTION
**Pull request by aperrin66**
Partly solves #39.
The summary field will be added when dealing with #43.

The base image for unit tests is updated to nansencenter/nansat_base:2.0.2.

There are also some minor fixes.

### CODE REVIEW

**Review by akorosov — APPROVED** [review]

**Comment by akorosov** [comment]
Thanks! Very good work!

File: metanorm/normalizers/url.py [hunk 1]
```diff
@@ -79,29 +80,26 @@ class URLMetadataNormalizer(BaseMetadataNormalizer):
...
+    urls_dataset_parameters = {
         'ftp://anon-ftp.ceda.ac.uk/neodc/esacci/sst/': ['sea_surface_temperature'],
-        'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L2.SST': [
-            'sea_surface_temperature'
-        ],
-        'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L3.SST_10': [
-            'sea_surface_temperature'
-        ],
-        'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2/L3.SST_25': [
+        'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2': [
```
Comment (aanersc) [inline metanorm/normalizers/url.py:102]: That 'ftp://ftp.gportal.jaxa.jp/standard/GCOM-W/GCOM-W.AMSR2' folder contains lots of things including sst, so it is incorrect to assign 'sea_surface_temperature' to all of them. 'sea_surface_temperature' is only for that three subfolders!

image

File: tests/normalizers/test_url.py [hunk 2]
```diff
@@ -139,6 +139,25 @@ def test_time_coverage_start_multiobs_glo_phy_nrt_015_003_hourly(self):
...
             datetime(year=2020, month=9, day=6, hour=0, minute=0, second=0, tzinfo=tzutc()))
 
+    def test_time_coverage_start_medsea_analysis_forecast_phy_006_013_day(self):
+        """Should return the proper starting time"""
+        urls = [
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-d/2020/06/20200601_d-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc',
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-h/2020/06/20200601_h-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc',
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-hts/2020/06/20200601_hts-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc'
+        ]
+        for url in urls:
```
Comment (aanersc) [inline tests/normalizers/test_url.py:149]: do not use for in the test, the test must be crystal clear in its assertions! please write them down individually.

File: tests/normalizers/test_url.py [hunk 3]
```diff
@@ -265,6 +284,25 @@ def test_time_coverage_end_multiobs_glo_phy_nrt_015_003_hourly(self):
...
             datetime(year=2020, month=9, day=7, hour=0, minute=0, second=0, tzinfo=tzutc()))
 
+    def test_time_coverage_end_medsea_analysis_forecast_phy_006_013_day(self):
+        """Should return the proper ending time"""
+        urls = [
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-d/2020/06/20200601_d-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc',
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-h/2020/06/20200601_h-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc',
+            'ftp://nrt.cmems-du.eu/Core/MEDSEA_ANALYSIS_FORECAST_PHY_006_013/med00-cmcc-cur-an-fc-hts/2020/06/20200601_hts-CMCC--RFVL-MFSeas5-MEDATL-b20200616_an-sv06.00.nc'
+        ]
+        for url in urls:
```
Comment (aanersc) [inline tests/normalizers/test_url.py:294]: No FOR for assertion!

**Review by aanersc — DISMISSED** [review]
Noted inline

File: metanorm/normalizers/url.py (hunk 1 above)
Comment (aperrin66) [reply metanorm/normalizers/url.py]: My bad, I removed that change

File: tests/normalizers/test_url.py (hunk 2 above)
Comment (aperrin66) [reply tests/normalizers/test_url.py]: done

File: tests/normalizers/test_url.py (hunk 3 above)
Comment (aperrin66) [reply tests/normalizers/test_url.py]: done

**Pull request merged**